docker-compose.yml

```diff
@@ -12,7 +12,7 @@ services:
       SECRET_KEY: fake
       DJANGO_SETTINGS_MODULE: wagtailio.settings.dev
       DATABASE_URL: postgres://wagtailio:wagtailio@db:5432/wagtailio
-
+      PRIMARY_HOST: http://localhost:8000
```

Comment:
note: #268 changes the healthcheck to use `pg_isready -U wagtailio` to avoid noise in the logs